src/features/ShowMovie/ui/MovieBlock.tsx

```diff
@@ -1,26 +1,46 @@
-import { FC } from 'react';
+import { FC, useState } from 'react';
+
+import { useParams } from 'react-router-dom';
+
+import { useGetMovie } from 'entities/movie/hooks/useGetMovie';
+import { LoadingSpinner } from 'shared/ui';
 
 import { DatesNav } from './DatesNav';
 import { MovieHeader } from './MovieHeader';
 import { ProjectionsList } from './ProjectionsList';
 
 export const MovieBlock: FC = () => {
+    const { id } = useParams();
+    const { data: movie, isLoading, isError } = useGetMovie(id!);
+    const [activeDate, setActiveDate] = useState<Date | null>(null);
+
+    if (isLoading) {
+        return <LoadingSpinner />;
+    }
+
+    if (isError) {
```

Comment:
It's not a first place where you check for an error and shows Error state (eg in MoviesList)
Think how you can make error state generic and reusable for all places where you need to show some error text